templates/components/vectordbs/typescript/llamacloud/generate.ts

```diff
@@ -8,16 +9,20 @@ import { checkRequiredEnvVars } from "./shared";
 dotenv.config();
 
 async function loadAndIndex() {
+  const index = await getDataSource();
+  const projectId = await index.getProjectId();
+  const pipelineId = await index.getPipelineId();
+  const llamaCloudFileService = new LLamaCloudFileService();
+
   const documents = await getDocuments();
```

Comment:
that's wrong. we shouldn't use the loaders (calling `getDocuments`), but directly upload the files to LlamaCloud (see Python code)